rules/windows/process_creation/proc_creation_win_reg_rdp_keys_tamper.yml

```
@@ -32,18 +46,23 @@ detection:
             - 'EnableConcurrentSessions'
     selection_values_2:
         CommandLine|contains:
-            - 'WinStations\RDP-Tcp'
-            - 'MaxInstanceCount'
+            - 'AllowTSConnections'
+            - 'fDenyTSConnections'
             - 'fEnableWinStation'
-            - 'TSUserEnabled'
-            - 'TSEnabled'
-            - 'TSAppCompat'
+            - 'fSingleSessionPerUser'
             - 'IdleWinStationPoolCount'
+            - 'MaxInstanceCount'
+            - 'SecurityLayer'
             - 'TSAdvertise'
-            - 'AllowTSConnections'
-            - 'fSingleSessionPerUser'
-            - 'fDenyTSConnections'
-    condition: all of selection_main_* and 1 of selection_values_*
+            - 'TSAppCompat'
+            - 'TSEnabled'
+            - 'TSUserEnabled'
+            - 'WinStations\RDP-Tcp'
+    filter_main_values_tls:
+        CommandLine|contains:
```

```suggestion
        CommandLine|contains|all:
```
Do you mean to add `|all` here, @nasbench?

---

Yes